test/c_api/connection_test.cpp

```diff
@@ -138,52 +138,52 @@ TEST_F(CApiConnectionTest, ExecuteError) {
     kuzu_prepared_statement_destroy(&preparedStatement);
 }
 
-TEST_F(CApiConnectionTest, QueryTimeout) {
-    kuzu_query_result result;
-    kuzu_state state;
-    auto connection = getConnection();
-    ASSERT_EQ(kuzu_connection_set_query_timeout(connection, 1), KuzuSuccess);
-    state = kuzu_connection_query(connection,
-        "MATCH (a:person)-[:knows*1..28]->(b:person) RETURN COUNT(*);", &result);
-    ASSERT_EQ(state, KuzuError);
-    ASSERT_NE(result._query_result, nullptr);
-    auto resultCpp = static_cast<QueryResult*>(result._query_result);
-    ASSERT_NE(resultCpp, nullptr);
-    ASSERT_FALSE(resultCpp->isSuccess());
-    ASSERT_EQ(resultCpp->getErrorMessage(), "Interrupted.");
-    kuzu_query_result_destroy(&result);
-    kuzu_connection badConnection;
-    ASSERT_EQ(kuzu_connection_init(nullptr, &badConnection), KuzuError);
-    ASSERT_EQ(kuzu_connection_set_query_timeout(&badConnection, 1), KuzuError);
-}
-
-#ifndef __SINGLE_THREADED__
-// The following test is disabled in single-threaded mode because it requires
-// a separate thread to run.
-TEST_F(CApiConnectionTest, Interrupt) {
-    kuzu_query_result result;
-    kuzu_state state;
-    auto connection = getConnection();
-    bool finished = false;
-
-    // Interrupt the query after 100ms
-    // This may happen too early, so try again until the query function finishes.
-    std::thread t([&connection, &finished]() {
-        do {
-            std::this_thread::sleep_for(std::chrono::milliseconds(100));
-            kuzu_connection_interrupt(connection);
-        } while (!finished);
-    });
-    state = kuzu_connection_query(connection,
-        "MATCH (a:person)-[:knows*1..28]->(b:person) RETURN COUNT(*);", &result);
-    finished = true;
-    ASSERT_EQ(state, KuzuError);
-    ASSERT_NE(result._query_result, nullptr);
-    auto resultCpp = static_cast<QueryResult*>(result._query_result);
-    ASSERT_NE(resultCpp, nullptr);
-    ASSERT_FALSE(resultCpp->isSuccess());
-    ASSERT_EQ(resultCpp->getErrorMessage(), "Interrupted.");
-    kuzu_query_result_destroy(&result);
-    t.join();
-}
-#endif
+// TEST_F(CApiConnectionTest, QueryTimeout) {
```

Comment:
What's the plan to uncomment this test? Will you merge in this commented way? If so, open an issue that will block the release. We should solve these issues before the release.

Reply:
Uncommented.